commands/issue.go

```diff
@@ -192,6 +202,7 @@ func listIssues(cmd *Command, args *Args) {
 			"creator":   flagIssueCreator,
 			"mentioned": flagIssueMentioned,
 			"labels":    flagIssueLabelsFilter,
+			"sort":      flagIssueSort,
```

Comment:
This always adds `sort: created` as the default parameter. We should verify this in one of the existing cukes by adding:

```
assert :sort => "created"
assert :direction => nil
```